pallets/staking/src/lib.rs

```diff
@@ -788,7 +792,7 @@ impl<BlockNumber: PartialEq> ElectionStatus<BlockNumber> {
     pub fn is_closed(&self) -> bool {
         match self {
             Self::Closed => true,
-            _ => false,
+            _ => false
```

Comment:
Could use std::matches!() here.

Reply:
The goal here is to minimize the diff to substrate [3.0.0](https://github.com/paritytech/substrate/blob/v3.0.0/frame/staking/src/lib.rs) so keeping as-is.